fitbenchmarking/utils/options.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, file_name=None):
         self.use_errors = fitting.getboolean('use_errors')
 
         plotting = config['PLOTTING']
+        self.make_plots = plotting.getboolean('make_plots')
```

Comment:
can we wrap this around a try-catch?  At the moment the code falls over with a cryptic error message if we send anything other than a boolean.

Reply:
A try-catch around `make_plots`? I'm not sure what you exactly mean and would we not have to do that with self.use_errors as well?

Reply:
I added a try-catch around `make_plots`, let me know what you think about the message